tests/library/test_json.py

```diff
@@ -1355,3 +1390,126 @@ def test_linked_list(self, target_obj):
 
         # The actual check
         generate_and_check(target_obj, schema_obj)
+
+
+class TestEmptySchemas:
+    empty_schema = "{}"
+    nested_empty_schema = """{
```

Comment:
Could you also do a case like
```json
{
  "properties" : {
    "a": {},
    "b": "number"
  },
  "type" : "object"
}
```

Reply:
Also, what if a `#def` has an empty schema (or a nested one)?

Reply:
Good ideas -- will do

Reply:
Added both of these tests!